data/json/items/ammo.json

```diff
@@ -285,9 +285,11 @@
         "ammo_type" : "BB",
         "damage" : 2,
         "range" : 12,
-        "dispersion" : 11,
-        "count" : 200,
-        "effects" : ["NEVER_MISFIRES"]
+        "dispersion" : 15,
```

Comment:
Multiplying 15 by itself will result in the most inherently inaccurate rifle ammo in the game, and it will interfere with my plan to introduce a pellet gun.